gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/VersionCompletionContributor.java

```diff
@@ -67,7 +70,14 @@ public void addCompletions(
                                                         .withLookupString("latest"),
                                                 Double.MAX_VALUE)
                                         : LookupElementBuilder.create(version))
-                                .forEach(resultSet::addElement);
+                                .toList();
+
+                        List<LookupElement> prioritizedVersions = IntStream.range(0, versions.size())
+                                .mapToObj(i ->
+                                        PrioritizedLookupElement.withPriority(versions.get(i), i - versions.size()))
+                                .toList();
```

Comment:
if you used `StreamEx` from line 64, you can actually just zip with an infinite stream that counts down. 

Doesn't really matter if it starts at 0 since you want relative order

e.g. 
```java
StreamEx(...)
  .flatMap(...)
  .zipWith(IntStream.iterate(0, i -> i - 1)
  .map((version, priority) -> getLookupElement(version, priority)
  .forEach(resultSet::addElement);
```